controllers/UsersController.js

```diff
@@ -1,92 +1,92 @@
-import { body, validationResult } from 'express-validator/check';
-import { sanitizeBody } from 'express-validator/filter';
-// import debug from 'debug';
+import Model from '../models/Model';
+import { InternalServerError } from '../utils/errorHandlers';
+// import { dev_logger } from '../utils/loggers';
 
-import users from '../utils/sample.users';
-
-// const logger = message => (debug('dev')(message));
+const users_model = new Model('users');
 
 const UsersController = {
 
-    dashboard: (req, res) => {
-        res.render( 'dashboard', { title: 'Dashboard' });
+    get_user: (req, res) => {
```

Comment:
Make this function async so you can do await within its body

Reply:
Done